teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name: 'unknown', parent_permission: true)
+    super(age, name, parent_permission)
```

Comment:
- [ ] Kindly use the colon notation to assign the `parent_permission` and `name` parameters for the super to avoid ArgumentError as you already use the colon notation in the parent class(Person), so it will be like:
```ruby
super(age, name: name, parent_permission: parent_permission)
```